script.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this eslint disable rule it's best to fix the issue in the code.